server/src/config-helper.ts

```diff
@@ -39,17 +37,17 @@ const DEFAULTS: CommonVoiceConfig = {
  * Create our configuration by merging config.json and our DEFAULTS.
  */
 export function getConfig(): CommonVoiceConfig {
-  const localConfig = load(LOCAL_CONFIG_PATH);
+  const localConfig = load();
   return Object.assign(DEFAULTS, localConfig);
 }
 
 /**
  * Attempt to load a json file, but return null if not found.
  */
-function load(path: string): CommonVoiceConfig {
+function load(): CommonVoiceConfig {
   let config = null;
   try {
-    config = require(path);
+    config = require('../config.json');
```

Comment:
I'm with you on that, I didn't have the patience to figure out why the import wouldn't work with an expression as path. Can look into it again

Reply:
possibly because if webpack might rewrite this path to refer to the same file while being in a different folder (i.e. where the output bundle is) -- which might only work if it's an inline literal.

pure guesswork though, would have to check this myself